subsystems/WikibaseRepo/02-Architecture Constraints.md

```diff
@@ -1,3 +1,5 @@
 # Architecture Constraints
 
+## [MediaWiki Extensions](./12-Glossary.md/#mediawiki-extension )
```

Comment:
As I tried to mention in https://github.com/wmde/wikidata-wikibase-architecture/pull/11#issuecomment-738823808 and https://github.com/wmde/wikidata-wikibase-architecture/pull/11#pullrequestreview-544988676 I don't agree that anything related to MediaWiki is an architectural constraint for the repo. I think anything related to MediaWiki on the repo part belongs in the "Architectural Decisions" section, not in the constraints.